geotrek/trekking/templates/trekking/sync_rando.html

```diff
@@ -0,0 +1,139 @@
+			    	    			  $("#progress-value").addClass('bar-success');
+			    	    		  }
+			    	    	  }
+			    	    	  if (this.result.infos) {
+			    	    		  $("#progress-text").text(this.result.infos); 
+			    	    	  }
+			    	      }
+			    	      else {
+			    	    	  if (this.status == 'FAILURE' && last_launched_tasked_id == this.id){
+			    	    		  // case of exception in task
+			    	    		  $("#progress-text").text("{% trans 'An error occured' %}");
+			    	    		  $('#exception-message').text(this.result.exc_type + ' : ' + this.result.exc_message)
+			    	    		  $("#progress-value").addClass('bar-danger');
+			    	    		  $("#progress-value").parent().removeClass("active");
+			    	    	  }
+			    	      }
+			    	  });
+			    	  
+			    	  if (!has_progress) {
+			    		  enable_sync_button();
+			    	  }
+			  });
+		}
+		
+		$(function(){
+			disable_sync_button(true);
+			get_sync_infos();
+			
+			$('#btn-confirm').click(function(evt) {
+				$("#progress-value").css("width", '0%');
+				$("#progress-text").text('');
+				$("#progress-value").parent().addClass("active");
+				$("#progress-value").removeClass('bar-success');
+				$("#progress-value").removeClass('bar-danger');
+				
+			    $.post(
+			        $('#form-sync').attr('action'), 
+			        $('#form-sync').serialize()
+		        );
+			    
+			    $('#confirm-submit').modal('toggle');
+			});
+			
+			window.setInterval(function(){
+				get_sync_infos();
+			}, 500);
+		});
+	</script>
+	<link rel="stylesheet" href="{% static 'trekking/css/sync_trek.css' %}" />
+{% endblock extrahead %}
+
+{% block toolbar %}
+{% endblock toolbar %}
+
+{% block mainpanel %}
+
+	<div id="sync-div" class="span3 offset2">
+	    <h3>{% trans "Trekking sync" %}</h3>
+		<div class="progress progress-striped active">
+		  <div id="progress-value" class="bar" style="width: 0%;"></div>
+		  <span id="progress-text"></span>
+		</div>
+		{% block mainform %}
+			{% crispy form form.helper %}
+		{% endblock mainform %}
+	</div>
+	
+	<div id="exception-message" style="display: none;">
+	</div>
+	
+	<div class="modal fade" id="confirm-submit" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+	    <!-- boostrap 2 modal confirmation -->
+	    <div class="modal-dialog">
+	        <div class="modal-content">
+	            <div class="modal-header">
+	                {% trans "Confirmation" %}
+	            </div>
+	            <div class="modal-body">
+	                {% trans "Are you sure you want to sync ?" %}
+	            </div>
+	
+	  			<div class="modal-footer">
+		            <button type="button" class="btn btn-default" data-dismiss="modal">
+		                {% trans "Cancel" %}
+		            </button>
+		            <a href="#" id="btn-confirm" class="btn btn-success success">
+		                {% trans "Sync" %}
+		            </a>
+		        </div>
+	    	</div>
+		</div>
+	</div>
```

Comment:
The confirmation modal is not required. Before there was no confirmation after click in menu, now there are 2 confirmations. 1 is sufficient.